tslint.json

```diff
@@ -10,6 +10,7 @@
         "no-internal-module": true,
         "no-trailing-whitespace": true,
         "no-var-keyword": true,
+        "no-unused-variable": true,
```

Comment:
Yea I actually wanted to talk about this a bit. Because these are linting errors, I wouldn't want our `gulp lint` task to pass if these problems existed in our code. I also would want my text editor to show these errors if it had `tslint`/linting integrated. This was why I ultimately added them to tslint instead of tsconfig.